api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -2551,6 +2556,34 @@ type AESCBCSpec struct {
 	BackupKey *corev1.LocalObjectReference `json:"backupKey,omitempty"`
 }
 
+type PayloadArchType string
+
+const (
+	AMD64   PayloadArchType = "AMD64"
+	PPC64LE PayloadArchType = "PPC64LE"
+	S390X   PayloadArchType = "S390X"
+	ARM64   PayloadArchType = "ARM64"
+	Multi   PayloadArchType = "MULTI"
```

Comment:
This isn't correct now, this should be pascal case to match the API enum (which is currently correct)